enterprise/tpa_pipeline.py

```diff
@@ -78,7 +98,10 @@ def active_provider_enforces_data_sharing(request, enforcement_location):
         enforcement_location (str): the point where to see data sharing consent state.
         argument can either be "optional", 'at_login' or 'at_enrollment'
     """
-    running_pipeline = request.session.get('partial_pipeline')
+    try:
+        running_pipeline = get_partial_pipeline(request)
+    except TypeError:
+        running_pipeline = request.session.get('partial_pipeline_token')
```

Comment:
@bradenmacdonald The issue is actually that the test code where this fails _is_ mocking out `get_partial_pipeline`.

Do you think it's okay to just leave it as `request.session.get('partial_pipeline_token')`? In this function we're just checking for whether the pipeline is running and nothing more, so what type is returned doesn't really matter as long as it indicates to us that the pipeline is indeed running, which the token should be enough for?